### PR TITLE
Add resources limits and requests for some charts

### PR DESCRIPTION
**Pull request by Ciello89**
Cleaning pull request #103

---
This change is [Reviewable](https://reviewable.io/reviews/att-comdev/openstack-helm/163)

### CODE REVIEW

**Comment by Ciello89** [comment]
Hey @alanmeadows, I think this pull request is good to go.

File: glance/templates/deployment-api.yaml [hunk 1]
```diff
@@ -35,6 +35,15 @@ spec:
         - name: glance-api
           image: {{ .Values.images.api }}
           imagePullPolicy: {{ .Values.images.pull_policy }}
+        {{- if .Values.resources.enabled }}
```
Comment (alanmeadows) [inline glance/templates/deployment-api.yaml]: @Ciello89 Looks like an indendation issue here - this should line up with L37.

File: glance/templates/deployment-api.yaml [hunk 2]
```diff
@@ -35,6 +35,15 @@ spec:
...
+          resources:
+            limits:
+              cpu: {{ .Values.resources.api.limits.cpu | quote }}
+              memory: {{ .Values.resources.api.limits.memory | quote }}
+            requests:
+              cpu: {{ .Values.resources.api.requests.cpu | quote }}
+              memory: {{ .Values.resources.api.requests.memory | quote }}
+        {{- end }}
```
Comment (alanmeadows) [inline glance/templates/deployment-api.yaml]: Same here - this should line up with L37.

File: glance/templates/deployment-api.yaml (hunk 1 above)
Comment (Ciello89) [inline glance/templates/deployment-api.yaml]: Done

File: glance/templates/deployment-api.yaml (hunk 2 above)
Comment (Ciello89) [inline glance/templates/deployment-api.yaml]: Done

File: memcached/templates/deployment.yaml [hunk 3]
```diff
@@ -23,6 +23,15 @@ spec:
         - name: memcached
           image: {{ .Values.images.memcached }}
           imagePullPolicy: {{ .Values.images.pull_policy }}
+          {{- if .Values.resources.enabled }}
+          resources:
+            limits:
+              cpu: {{ .Values.resources.api.limits.cpu }}
```
Comment (larryrensing) [inline memcached/templates/deployment.yaml]: for consistency you should add the " {{ ... | quote }} " here

Comment (larryrensing) [inline memcached/templates/deployment.yaml]: for consistency you should add {{ .... | quote }} in memcached also

Comment (alanmeadows) [inline memcached/templates/deployment.yaml]: Agreed

**Pull request merged**